enterprise/admin/__init__.py

```diff
@@ -15,9 +15,9 @@
 from enterprise.admin.utils import UrlNames
 from enterprise.admin.views import EnterpriseCustomerManageLearnersView
 from enterprise.django_compatibility import reverse
-from enterprise.models import (
+from enterprise.models import (  # pylint:disable=no-name-in-module
     EnterpriseCustomer, EnterpriseCustomerUser, EnterpriseCustomerBrandingConfiguration,
-    EnterpriseCustomerIdentityProvider,
+    EnterpriseCustomerIdentityProvider, HistoricalUserDataSharingConsentAudit
```

Comment:
The magic that automatically inserts `HistoricalUserDataSharingConsentAudit` into `models` is rather ugly.  I wasn't aware it was doing that, so this import looked a bit surprising to me.  Why do you need `HistoricalUserDataSharingConsentAudit` rather than `UserDataSharingConsentAudit` here?